cmd/tusd/cli/flags.go

```diff
@@ -89,3 +105,31 @@ func ParseFlags() {
 			"Please remove underscore from the value", Flags.GCSObjectPrefix)
 	}
 }
+
+func SetEnabledHooks() {
+	if Flags.EnabledHooks == "*" {
+		for _, h := range hooks.AvailableHooks {
+			EnabledHooks = append(EnabledHooks, h)
+		}
+	} else {
+		slc := strings.Split(Flags.EnabledHooks, ",")
```

Comment:
It would be nice to have a check here if the specified hook name actually exists, so users get notified early when they misspell `post-receive` as `post-recieve`, for example.